fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStoreTestBase.java

```diff
@@ -146,34 +175,14 @@ protected RecordLayerPropertyStorage.Builder addDefaultProps(RecordLayerProperty
         return props;
     }
 
-    @BeforeEach
-    public void clearAndInitialize() {
-        getFDB();
-        clear();
-
-        // Reset these indexes added and last modified versions, which can be updated during tests.
-        // For example, adding the indexes to a RecordMetaDataBuilder can update these fields
-        COUNT_INDEX.setAddedVersion(0);
-        COUNT_INDEX.setLastModifiedVersion(0);
-        COUNT_UPDATES_INDEX.setAddedVersion(0);
-        COUNT_UPDATES_INDEX.setLastModifiedVersion(0);
-    }
-
-    protected void clear() {
-        fdb.run(timer, null, context -> {
-            path.deleteAllData(context);
-            return null;
-        });
-    }
-
-    @AfterEach
-    public void checkForOpenContexts() {
-        int count = fdb.warnAndCloseOldTrackedOpenContexts(0);
-        assertEquals(0, count, "should not have left any contexts open");
```

Comment:
This `checkForOpenContexts` check has been moved to the `FDBDatabaseExtensions`'s `afterEach` callback.